cli/src/main/kotlin/OrtMain.kt

```diff
@@ -80,7 +81,11 @@ data class GlobalOptions(
  */
 fun main(args: Array<String>) {
     Os.fixupUserHomeProperty()
-    OrtMain().main(args)
+    try {
+        OrtMain().main(args)
+    } finally {
+        println("ORT command execution terminated at ${LocalDateTime.now()}.")
```

Comment:
Can we make this a `debug` log instead? Like

    val ortMain = OrtMain()
    try {
        ortMain.main(args)
    } finally {
        ortMain.logger.debug { ... }
    }

Reply:
Done. Then it is not necessary to add the time to the message, since this should be part of the log output anyway.